### ISSUE TITLE
Npm package

### ISSUE THREAD

**Issue by galvakojis** [gildas-lormeau/zip.js#344]
I am not sure how it works, but could you update this package from npm?
https://www.npmjs.com/package/zip-js

it took me some time to realize that I have an old version

**Comment by 0f-0b** [comment]
The latest version is available at <https://www.npmjs.com/package/@zip.js/zip.js>.

**Comment by gildas-lormeau** [comment]
Indeed, I'm not the person who published https://www.npmjs.com/package/zip-js which is the legacy version of zip.js, maybe @johanpoirier could remove it from NPM to fix this?

**Comment by galvakojis** [comment]
some note or warning would be enough, cause at the moment it can be confusing 
![image](https://user-images.githubusercontent.com/17315578/182099427-e78642bb-cde0-4900-ada0-548e8229099d.png)

**Comment by gildas-lormeau** [comment]
@johanpoirier It looks like you updated your README. However, you have to bump your version an publish it in order to update the README on NPM.
